x-pack/plugins/cases/docs/typedoc.json

```diff
@@ -0,0 +1,25 @@
+{
+  "entryPoints": [
+    "../server/client/client.ts",
+    "../server/client/typedoc_interfaces.ts",
+    "../server/client/attachments",
+    "../server/client/cases/client.ts",
+    "../server/client/cases/get.ts",
+    "../server/client/cases/push.ts",
+    "../server/client/configure/client.ts",
+    "../server/client/stats/client.ts",
+    "../server/client/sub_cases/client.ts",
+    "../server/client/user_actions/client.ts"
+  ],
+  "exclude": [
+    "**/mock.ts",
+    "../server/client/cases/+(mock.ts|utils.ts|utils.test.ts|types.ts)"
+  ],
+  "excludeExternals": true,
```

Comment:
The `IAllCommentsResponse` doesn't get rendered correctly and typedoc thinks it an extension of an array and includes every field/method on an array. Excluding externals keeps it from doing that.